### DISCUSSION TITLE
Addon broken?

### DISCUSSION THREAD

**Discussion by hillshome** [fuatakgun/eufy_security#1078] [Q&A]
I keep getting addon broken notifications when starting the eufy cs addon. ![image](https://github.com/fuatakgun/eufy_security/assets/127354307/fbd1c707-493e-4f73-ad22-6d99c70b9392)

**Comment by fuatakgun** [comment]
Yes, connection is broken and integration is trying to reconnect.

**Reply by hillshome** [reply]
Is it a update to fix? Also my streams stop after couple a minutes.
